rainforest-cli.go

```diff
@@ -1,83 +1,304 @@
+				},
+				cli.StringFlag{
+					Name:  "import-variable-name",
+					Usage: "`NAME` of the tabular variable to be created or updated.",
+				},
+				cli.StringFlag{
+					Name:  "import-variable-csv-file",
+					Usage: "`PATH` to the CSV file to be uploaded.",
+				},
+				cli.BoolTFlag{
+					Name:  "overwrite-variable",
+					Usage: "If the flag is set, named variable will be updated.",
+				},
+				cli.BoolTFlag{
+					Name:  "single-use",
+					Usage: "This option marks uploaded variable as single-use",
+				},
+			},
+		},
+		{
+			Name:      "new",
+			Usage:     "Create a new RFML test",
+			ArgsUsage: "[name]",
+			Description: "Create new Rainforest test in RFML format (Rainforest Markup Language). " +
+				"You may also specify a custom test title or file name.",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:   "test-folder",
+					Value:  "./spec/rainforest/",
+					Usage:  "`PATH` at which to create new test.",
+					EnvVar: "RAINFOREST_TEST_FOLDER",
+				},
+			},
+			Action: notImplemented,
+		},
+		{
+			Name:      "validate",
+			Usage:     "Validate your RFML tests",
+			ArgsUsage: " ",
+			Description: "Validate your tests for syntax and correct RFML ids for embedded tests. " +
+				"If API token is set it'll validate your tests against server data as well.",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:   "test-folder",
+					Value:  "./spec/rainforest/",
+					Usage:  "`PATH` where to look for a tests to validate.",
+					EnvVar: "RAINFOREST_TEST_FOLDER",
+				},
+			},
+			Action: notImplemented,
+		},
+		{
+			Name:      "upload",
+			Usage:     "Upload your RFML tests",
+			ArgsUsage: "[path to RFML file]",
+			Description: "Uploads specified test to Rainforest. " +
+				"If no filepath is given it uploads all RFML tests.",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:   "test-folder",
+					Value:  "./spec/rainforest/",
+					Usage:  "`PATH` where to look for a tests to upload.",
+					EnvVar: "RAINFOREST_TEST_FOLDER",
+				},
+			},
+			Action: notImplemented,
+		},
+		{
+			Name:        "rm",
+			Usage:       "Remove an RFML test locally and remotely",
+			ArgsUsage:   "[path to RFML file]",
+			Description: "Remove RFML file and remove test from Rainforest test suite.",
+			Action:      notImplemented,
+		},
+		{
+			Name: "download",
+			// Left for legacy reason, should nuke?
+			Aliases:   []string{"export"},
+			Usage:     "Download your remote Rainforest tests to RFML",
+			ArgsUsage: "[test IDs]",
+			Description: "Download your remote tests from Rainforest to RFML. " +
+				"You need to specify list of test IDs to download or use keyword 'all'. " +
+				"Alternatively you can use one of the filtering options.",
+			Flags: []cli.Flag{
+				cli.StringSliceFlag{
+					Name:  "tag",
+					Usage: "filter tests by `TAG`. Can be used multiple times for filtering by multiple tags.",
+				},
+				cli.StringFlag{
+					Name:  "site, site-id",
```

Comment:
I saw a few commands that now have multiple aliases. We should keep the README up to date as we change things so we don't lose track maybe?